iib/workers/tasks/utils.py

```diff
@@ -120,11 +123,13 @@ def run_cmd(cmd, params=None, exc_msg=None):
     params.setdefault('stderr', subprocess.PIPE)
     params.setdefault('stdout', subprocess.PIPE)
 
-    log.debug('Running the command "%s"', cmd)
+    cmd_repr = cmd_repr or cmd
```

Comment:
Could you move this to the beginning of the function? I like to avoid reassigning argument values, but if it's needed, then I like it at the top of the function to make it clear that the arguments don't change after that point.